_posts/2018-10-29-help-test-rust-2018.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Help test Rust 2018"
+---
+
+Back in July, we talked about ["Rust 2018"]. In short, we're adding a new
+concept to Rust, "Editions." Editions are a way of showing larger progress
```

Comment:
I would try to phrase this second sentence differently. As written, "new concept" sounds heavy and sounds like something that increases complexity: "we are adding a new concept: inheritance!" or "we are adding a new concept: monads! Monads are a way of showing larger progress in our grasp of programming!". I would phrase this to feel lighter and deliberate and forward-looking e.g. my quick attempt:

> In short, we are launching a cycle of long-term milestones called "Editions". Editions are a way to capture the progress delivered incrementally by our ordinary six-week release cycle -- and focus Rust libraries, tooling, and documentation cohesively around it. Editions will be selected roughly every three years: Rust 1.0 was "Rust 2015" and Rust 1.31 will be "Rust 2018".

Reply:
We should also mention something about breakage here. That was indeed covered in the previous post, but I don't think that's enough.